platform/jobs/edxPlatformQualityMaster.groovy

```diff
@@ -33,7 +33,7 @@ archiveReports += 'edx-platform*/test_root/log/*.log, edx-platform*/test_root/lo
 archiveReports += 'edx-platform*/**/nosetests.xml,edx-platform*/**/TEST-*.xml'
 
 String htmlReports = 'pylint/*view*/, pep8/*view*/, jshint/*view*/, python_complexity/*view*/,'
-htmlReports += 'safecommit/*view*/, safelint/*view*/, eslint/*view*/'
+htmlReports += 'xsscommitlint/*view*/, xsslint/*view*/, eslint/*view*/'
```

Comment:
We should support both lint-file paths for a while (to not break existing PRs or require that everyone rebase). In a week or two, we can remove the old-style path